ui/app/document-upload/views/documentUpload.html

```diff
@@ -1,37 +1,36 @@
 <div id="messages" class="messages">
-    <div class="info-message" ng-show="success"><p>File(s) updated successfully</p></div>
-    <div class="error-message" ng-show="orientation_warning"><p><i class="fa fa-exclamation-sign"></i>The device is not
-        held upright. Please rotate the device before taking photo</p></div>
+    <div class="info-message" ng-show="success"><p>{{:'FILE_UPLOAD_SUCCESS_MESSAGE'|translate }}</p></div>
+    <div class="error-message" ng-show="orientation_warning"><p><i class="fa fa-exclamation-sign"></i>{{:'PLEASE_ROTATE_DEVICE_MESSAGE'|translate }}</p></div>
 </div>
 <div ng-if="toggleGallery">
     <ul class="doc-upload-accordion" bm-gallery patient="patient" access-impression="false">
         <li id="newVisit">
             <h5 ng-click="resetCurrentVisit(newVisit)">
                 <i ng-hide="isCurrentVisit(newVisit)" class="fa fa-plus-circle state"></i>
                 <i ng-if="isCurrentVisit(newVisit)" class="fa fa-minus-circle state"></i>
-                Visit: New
+                {{:'NEW_VISIT_LABEL'|translate }}
```

Comment:
`{{:` does not work. I believe the syntax is `{{::`

Reply:
oh yes, Thanks @rbuisson . will be updated accordingly.